cases/management/commands/import_ciaa_cases.py

```
@@ -0,0 +1,191 @@
+
+                total_created += created
+                total_skipped += skipped
+                total_failed += failed
+
+            self._log_summary(total_created, total_skipped, total_failed, dry_run)
+
+            # Exit with error code if any imports failed
+            if total_failed > 0:
+                raise CommandError(f"Import completed with {total_failed} failures")
+
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)